File: f77/build03/term.f
C
       Subroutine Termination_Conditions(Quit)
C
C      Termination module
C
C      This module determines if the run termination conditions have been met
C
C      Inputs:
C        Time - Simulation time [sec]
C        Tatpr = Simulation termination time [sec]
C
C      Outputs:
C        Quit - Stop simulation run [Boolean]
C
C      Internal variables and constants:
C        None.
C
C      Declare global common and assign variable locations
C
      Include '../driver/global.inc'
      Include '../driver/sysvars.inc'
      Logical Quit
C
      Quit = Time  .ge.  Tstop
C
      Return
      End

File: compcert/build03/term.c
#include "term.h"
#include "../driver/global.h"
#include "../driver/sysvars.h"
BOOL termination_conditions(void) {
  // This module determines if the run termination conditions have been met
  return time >= tstop ? TRUE : FALSE;  // Stop simulation run [Boolean]
}
